README.md

```diff
@@ -14,21 +14,36 @@ An opinioned coding guideline for React + Redux project @WORD UP
 
 ## File Structure
   * Use `use-cases` to contains the files related to React & Redux
+  * Reference:
+    * [Tips For a Better Redux Architecture: Lessons for Enterprise Scale](https://hashnode.com/post/tips-for-a-better-redux-architecture-lessons-for-enterprise-scale-civrlqhuy0keqc6539boivk2f)
+    * [Architecture the Lost Years by Robert Martin](https://www.youtube.com/watch?v=WpkDN78P884)
   * `use-cases/`
     * `user-profile/`
       * `index.js` => might be a container
```

Comment:
I don't really understand the comment "might be a container" mean?
what's the definition about 'container' here?